src/zombies.test.ts

```diff
@@ -1,30 +1,63 @@
 import { ok } from "node:assert/strict";
 import { test } from "node:test";
 
-const createRoom = (capacity: number) => {
-  const _capacity = capacity;
+const createRoom = (initCapacity: number) => {
+  let _capacity = initCapacity;
```

Comment:
Capacity is meant to be constant as it's the room's capacity; the number of zombies the room can hold.